explainer.md

```diff
@@ -121,10 +125,9 @@ transition element
 - **image wrapper**: This element has a width and height of 100%, and [`isolation: isolate`](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation). This wrapper is useful when cross-fading images (documented later).
 - **image**: This contains the cached image, which may paint outside the parent elements. This would be a replaced element so CSS properties like `object-fit` will be supported. This element has a width and height of 100%, although the image may paint outside of its own bounds, similar to how a `box-shadow` is painted outside of an element's bounds.
 
-These elements will be addressable via pseudo-elements, although they may be exposed as full elements via a JS API.
+These elements will be addressable via pseudo-elements.
```

Comment:
not sure what "addressable" means in this context, but this was there before

Reply:
How about now? The aim is to say that developers can access them as pseudo elements.